testing/src/components/container/Item.js

```diff
@@ -5,6 +5,62 @@
 
 import React from "react";
 
-export const Item = () => {
- return <div>NewFile</div>;
+const MakeList = (props) => {
+
+    return props.items.map(items => 
+        <div key={items.id} style={{
+            border:"solid 1px black",
```

Comment:
style in css not inline, it messed with cascading

Reply:
fixed!